packages/wallet/ui/src/util/WalletBackendAdapter.js

```diff
@@ -130,15 +130,80 @@ export const makeWalletBridgeFromFollower = (
     ]),
   );
 
+  // We assume just one cosmos purse per brand.
```

Comment:
could ref https://github.com/Agoric/agoric-sdk/issues/6126 (though I now see it is below)